scipy/stats/_morestats.py

```diff
@@ -4105,6 +4105,10 @@ def wilcoxon(x, y=None, zero_method="wilcox", correction=False,
     WilcoxonResult(statistic=6.0, pvalue=0.4375)
 
     """
+    return _wilcoxon._wilcoxon_nd(x, y, zero_method, correction, alternative, method)
+
+
+def _wilcoxon_1d(x, y, zero_method, correction, alternative, method):
```

Comment:
should this be deleted before we merge this PR. It is no longer used

Reply:
Sure. Did you see the test at the top post where I used this to confirm that the behavior (in including warnings) is the same for all combinations of arguments? That's what I left it for, anyway, but if you're done with that, we can remove it

Reply:
yes, I saw that. I just wanted to confirm it is not needed. I saw you already removed it.